### PR TITLE
Improve/use vote counts stored in model

### PR DESCRIPTION
**Pull request by MauriceNaftalin**

### CODE REVIEW

File: src/main/java/com/voteroid/messagingstompwebsocket/PresentationController.java [hunk 1]
```diff
@@ -64,4 +74,4 @@ public Slide getSlide(@PathVariable String questionName) {
     // alternative annotations (not sure of the difference)
     // @MessageMapping("/vote") [answer: used to receive STOMP messages sent with prefix set by setApplicationDestinationPrefixes]
     // @SendTo("/topic/votes")
-}
+}
```
Comment (lifey) [inline src/main/java/com/voteroid/messagingstompwebsocket/PresentationController.java:77]: github is annoyed when a file is not terminated with a \n

File: src/main/java/com/voteroid/messagingstompwebsocket/PresentationController.java [hunk 2]
```diff
@@ -20,15 +19,26 @@ public class PresentationController {
...
+    public void acceptVote(int answerIndex) {
+        logger.debug("acceptVote: " + answerIndex);
+        final Slide currentSlide = presentation.getCurrentSlide();
+        // answerIndex is 1-indexed, and so is d3
+        final int voteCount = currentSlide.answers().get(answerIndex - 1).votes().incrementAndGet();
+        simpTemplate.convertAndSend("/topic/votes/" + currentSlide.question(),
+                Map.of("content", Map.of("answerIndex", answerIndex, "voteCount", voteCount)));
+    }
+
+/*
```
Comment (lifey) [inline src/main/java/com/voteroid/messagingstompwebsocket/PresentationController.java:32]: As you have full history no need to comment out old code

File: examples/presentation-local.adoc [hunk 3]
```diff
@@ -38,7 +38,7 @@ System.out.print(intList.contains(1));
 [.column]
 [pass]
 ++++
-<div id="question1">x</div>
+<div id="question1">No connection to server</div>
```
Comment (lifey) [inline examples/presentation-local.adoc:41]: this is replaced with content ? so it is like alt-text ?

File: src/main/resources/static/iframeStompManager.js [hunk 4]
```diff
@@ -11,8 +11,8 @@ stompClient.onConnect = (frame) => {
     console.log('Connected: ' + frame);
     const params = new URLSearchParams(window.location.search);
     const question = params.get('question');
-    stompClient.subscribe('/topic/vote/' + question, (vote) => {
-        increaseBarHeight(JSON.parse(vote.body).content);
+    stompClient.subscribe('/topic/votes/' + question, (voteUpdate) => {
+        setBarHeight(JSON.parse(voteUpdate.body).content);
```
Comment (lifey) [inline src/main/resources/static/iframeStompManager.js:15]: much better 👍

**Review by lifey — CHANGES_REQUESTED** [review]

File: src/main/java/com/voteroid/messagingstompwebsocket/PresentationController.java (hunk 1 above)
Comment (MauriceNaftalin) [reply src/main/java/com/voteroid/messagingstompwebsocket/PresentationController.java:77]: 👍

File: src/main/java/com/voteroid/messagingstompwebsocket/PresentationController.java (hunk 2 above)
Comment (MauriceNaftalin) [reply src/main/java/com/voteroid/messagingstompwebsocket/PresentationController.java:32]: 👍

File: examples/presentation-local.adoc (hunk 3 above)
Comment (MauriceNaftalin) [reply examples/presentation-local.adoc:41]: If the presentation is opened without a server running, this is the text that appears in place of the iframe. Once the server is running (and the page is refreshed), the iframe appears instead.

File: src/main/resources/static/iframeStompManager.js (hunk 4 above)
Comment (MauriceNaftalin) [reply src/main/resources/static/iframeStompManager.js:15]: 😀

**Review by lifey — APPROVED** [review]

**Pull request merged**